projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -1072,15 +1072,24 @@ public static function set_availability_for_plan( $slug ) {
 		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
 		$is_atomic_site = ( new Host() )->is_woa_site();
 
+		// keep track of specific features data per blog between calls.
+		static $site_specific_features = array();
```

Comment:
I think it makes sense to move this definition to the class level - as we did in #22916.